packages/ui/src/Tooltip/Tooltip.tsx

```diff
@@ -1,20 +1,12 @@
-import React from "react";
-import { default as RcTooltip } from "rc-tooltip";
-import "rc-tooltip/assets/bootstrap_white.css";
-import "./style.scss";
-
-type TooltipPropsContent =
-    | (() => React.ReactChild)
-    | React.ReactChild
-    | React.ReactFragment
-    | React.ReactPortal;
+import React, { useCallback } from "react";
+import { Tooltip as TooltipAdmin, TooltipProps as TooltipPropsAdmin } from "@webiny/admin-ui";
 
 export interface TooltipProps {
     // A component (eg. button) which will trigger the tooltip.
     children: React.ReactNode;
```

Comment:
I'm assuming you did not put deprecation notices on props since the whole component is marked as deprecated, right?

Reply:
Yes, correct 👍